Tests/Configuration/appConfig.json

```
@@ -2,5 +2,7 @@
     "url": "",
     "rnwAdminUrl": "",
     "username":"",
```

Can refer username as B2C user.
```suggestion
    "B2CAutoTest_User":"",
```

---

Variable name changed